src/content/developers/tutorials/erc20-annotated-code/index.md

```diff
@@ -468,7 +470,7 @@ strings, can be stored in three locations:
 |           | Lifetime      | Contract Access | Gas Cost |
 | --------- | ------------- | --------------- | -------- |
 | Memory    | Function call | Read/Write      | Tens or hundreds (higher for higher locations) |
-| Calldata  | Function call | Read Only       | Very low (paid by the caller, not the callee)  | 
+| Calldata  | Function call | Read Only       | Irrelevant, can only be used for input to the contract | 
```

Comment:
Just double checking, is "can only be used for input to the contract" worded right? Is this used to input data? Or retrieve data?

Reply:
Thank you for noticing, it really wasn't clear enough. I replaced it with:

> Can't be used as a return type, only a function parameter type